src/components/ListingComponent.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import { getUser } from '../api';
+
+const ListingComponent = (props) => {
+    const [user, setUser] = useState([]);
+
+    useEffect(() => {
+        try {
+            const temp = getUser(props.seller_id);
```

Comment:
If there's 50 total listings, then this will make 50 network requests. I'd say the backend should just include the seller's name/email in each listing from `/getListings` instead of making the frontend fetch it all on its own. Adjusting the server query to use `JOIN` should do the trick.